web-ui/controller/git_forge.ml

```diff
@@ -97,9 +98,16 @@ module Make (View : View) = struct
     Client.Commit.refs commit >>!= fun refs ->
     let csrf_token = Dream.csrf_tag request in
     let flash_messages = Dream.flash_messages request in
-    Dream.respond
-    @@ View.list_steps ~org ~repo ~refs ~hash ~jobs ~csrf_token ~flash_messages
-         ()
+    let first_step_queued_at = Run_time.first_step_queued_at jobs in
+    match first_step_queued_at with
+    | Error e ->
+        Dream.log "Error - %s" e;
+        Dream.empty `Internal_Server_Error
```

Comment:
I got this error when rebuilding a step. I was wondering how much it could be an issue.

Reply:
That's interesting - I intentionally had it error out because I want to know the circumstances in which this can happen. 

I guess if everything is in the state `Not Started` it could. If it happens routinely then of course we should not crash. I'd like to "test this in prod" for a bit to better understand this. Can you recall the state of your system when you noticed the error?

Reply:
On further reflection I am going to change my mind and not throw an error here. I don't think it's appropriate to crash if we can't get a timestamp.